lib/processors/bundlers/flexChangesBundler.js

```diff
@@ -20,9 +20,10 @@ import {createResource} from "@ui5/fs/resourceFactory";
  * @param {string} parameters.options.pathPrefix Prefix for bundle path
  * @param {string} parameters.options.hasFlexBundleVersion true if minUI5Version >= 1.73 than
  *															create flexibility-bundle.json
+ * @param {object} parameters.existingFlexBundle
```

Comment:
Please add a description for this new parameter

Reply:
Added description